comp/core/tagger/global.go

```diff
@@ -44,8 +44,25 @@ func GetEntity(entityID types.EntityID) (*types.Entity, error) {
 	return globalTagger.GetEntity(entityID)
 }
 
+// LegacyTag is an interface function that queries taggerclient singleton
+// If possible, avoid using this function, and use the Tag interface function instead.
+// This function exists in order not to break backward compatibility with rtloader and python
+// integrations using the tagger
+func LegacyTag(entity string, cardinality types.TagCardinality) ([]string, error) {
+	if globalTagger == nil {
+		return nil, fmt.Errorf("a global tagger must be set before calling Tag")
```

Comment:
`fmt.Errorf` is a function aiming at formatting a string.
When there’s nothing to format, it’s much more efficient to use a function that just wraps the already formatted string into an `error`:

```suggestion
		return nil, errors.New("a global tagger must be set before calling Tag")
```

Reply:
Indeed you are right. 

fmt.Errorf is already used in this file in many place, where it can be replaced by `errors.New`. 

I updated all of them.

Reply:
Indeed, I noticed that this pattern was common in our code base.
I’ve also noticed that the [`perfsprint` linter](https://github.com/catenacyber/perfsprint), [included in latest versions of `golangci-lint`](https://golangci-lint.run/usage/linters/#perfsprint) [complains about it](https://golangci-lint.run/usage/linters/#perfsprint).
So, we’ll hopefully have to fix all of them next time we update `golangci-lint`.